api/src/main/java/org/openmrs/module/cameroonappointmentresponseextension/extensions/EmergencyContactDetails.java

```diff
@@ -0,0 +1,32 @@
+package org.openmrs.module.cameroonappointmentresponseextension.extensions;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import org.openmrs.PersonAttributeType;
+import org.openmrs.api.context.Context;
+import org.openmrs.module.appointments.model.Appointment;
+import org.openmrs.module.appointments.web.extension.AppointmentResponseExtension;
+import org.springframework.stereotype.Component;
+
+@Component
+public class EmergencyContactDetails implements AppointmentResponseExtension {
```

Comment:
@rubailly We should return more than just EmergencyContactDetails here right? I would call this class AdditionalDetails

Reply:
Agree. Especially since we are adding Patient's phone number and village, EmergencyContactDetails details wouldn't be the right name.

I'll rename the class to "AdditionalDetails" and send back to you.